app/utils/cache-control.test.ts

```diff
@@ -3,80 +3,78 @@ import { describe, it, expect } from 'vitest'
 import { parse, CacheControl } from './cache-control'
 
 describe('parse', () => {
-  it('parses basic cache control directives', () => {
+  it('parses max-age directive from cache control header', () => {
     const result = parse('max-age=3600, public')
 
     expect(result.maxAge).toBe(3600)
     expect(result.public).toBe(true)
```

Comment:
This test claims to be about MaxAge, but it's testing both MaxAge and public. Please split the test. This test claims to be about MaxAge, but it's testing both MaxAge and public. Please split the test.